demos/security_barrier_camera_demo/main.cpp

```diff
@@ -810,30 +844,32 @@ int main(int argc, char* argv[]) {
         }
 
         // Running
-        context.t0 = std::chrono::steady_clock::now();
         worker->runThreads();
+        context.t0 = std::chrono::steady_clock::now();
```

Comment:
This seems suspicious. Why did you do this?

Reply:
The idea was to remove overhead on thread spawning, but since a `Drawer` now reads this var, I should put it back.

Reply:
reverted